docs/components/MessagingThreadHistoryView.jsx

```diff
@@ -65,8 +65,10 @@ export default class MessagingThreadHistoryView extends React.PureComponent {
         sourcePath="src/MessagingThreadHistory/MessagingThreadHistory.tsx"
       >
         <header className={cssClass.INTRO}>
-          <p>TODO: Describe your component/state it's purpose</p>
-          <p>TODO(optional): Describe scenarios where the component might be useful.</p>
+          <p>
+            MessagingThreadHistoryView is a container for messages between users, in the form of
```

Comment:
Figured a description would be good here.

Reply:
Damn I got sloppy with my components PR 😅  Thanks for fixing my many tiny mistakes 🙏 

Nit: `MessagingThreadHistoryView` -> `MessagingThreadHistory`

Reply:
I find it so appropriate that I made a silly oversight while fixing this, hah.